backend/src/packages/drivers/driver.entity.ts

```diff
@@ -12,42 +13,56 @@ class DriverEntity implements IEntity {
 
   private businessId: DriverEntityT['businessId'];
 
+  private createdAt: DriverEntityT['createdAt'];
+
+  private user?: UserEntityT;
```

Comment:
why user is optional here?

Reply:
Because  when we create a new driver we can’t  set a user to initializeNew  (type UserEntityT and type Omit<UserEntity, "groupId" | "passwordHash" | "passwordSalt">), if we change the type to Omit<UserEntity, "groupId" | "passwordHash" | "passwordSalt"> we get a new problem with conversation users that we get from db.